src/panels/config/integrations/integration-panels/matter/matter-add-device.ts

```diff
@@ -1,13 +1,15 @@
+import { LitElement } from "lit";
 import { customElement } from "lit/decorators";
 import { navigate } from "../../../../../common/navigate";
 import { HomeAssistant } from "../../../../../types";
 import { showMatterAddDeviceDialog } from "./show-dialog-add-matter-device";
 
 @customElement("matter-add-device")
-export class MatterAddDevice extends HTMLElement {
+export class MatterAddDevice extends LitElement {
```

Comment:
I'm not sure why it's HTMLElement, but the linting introduced in https://github.com/home-assistant/frontend/pull/21940 forces us to use LitElement. We should revert that one if this is inappropiate.

Reply:
It is because the `customElement` decorator is used, that was added in that release of es-lit, which I don't agree with that but ok...

This is `HTMLElement` because it is not needed to be a `LitElement`, because it doesnt use Lit (`LitElement` extends `ReactiveElement`) or properties it need to be updated from (`ReactiveElement` extends `HTMLElement`)

We can either remove the `customElement` decorator, and register the element manually:
```js
customElements.define("matter-add-device", MatterAddDevice);
```

Or ignore the linting issue.